yb-voyager/cmd/common.go

```diff
@@ -1129,27 +1129,37 @@ type AssessMigrationDBConfig struct {
 // =============== for yugabyted controlplane ==============//
 // TODO: see if this can be accommodated in controlplane pkg, facing pkg cyclic dependency issue
 
-var ASSESS_MIGRATION_YBD_PAYLOAD_VERSION = "1.1" // version(s) till now: 1.0, 1.1
+/*
+Version History
+1.0: Introduced AssessmentIssue field for storing assessment issues in flattened format
+1.1: Added TargetDBVersion and AssessmentIssueYugabyteD.MinimumVersionFixedIn
+1.2: Syncing it with original AssessmentIssue(adding fields Category, CategoryDescription, Type, Name, Description, Impact, ObjectType) and MigrationComplexityExplanation;
+*/
+var ASSESS_MIGRATION_YBD_PAYLOAD_VERSION = "1.2"
 
 type AssessMigrationPayload struct {
-	PayloadVersion        string
-	VoyagerVersion        string
-	TargetDBVersion       *ybversion.YBVersion
-	MigrationComplexity   string
-	SchemaSummary         utils.SchemaSummary
-	AssessmentIssues      []AssessmentIssueYugabyteD
-	SourceSizeDetails     SourceDBSizeDetails
-	TargetRecommendations TargetSizingRecommendations
-	ConversionIssues      []utils.AnalyzeSchemaIssue
+	PayloadVersion                 string
+	VoyagerVersion                 string
+	TargetDBVersion                *ybversion.YBVersion
+	MigrationComplexity            string
+	MigrationComplexityExplanation string
```

Comment:
How are we populating `MigrationComplexityExplanation`, don't see any change related to that here?

Reply:
OR are we just adding the field but not sending the actual explanation as of this PR?

Reply:
oh right, that was a miss. Updated.